packages/api-specification/generate-docs.js

```diff
@@ -70,6 +70,12 @@ const generateInterface = (interfaceObj) => {
 const generateProperty = (prop, className) => {
   const typeArgsString = getType(prop.type);
   mdObject[className] += `**${prop.name}${prop.flags.isOptional ? '?' : ''}**: \`${typeArgsString}\`  \n`;
+  const comment = prop.comment;
+  if (comment) {
+    if (comment.shortText) {
```

Comment:
Yes, although I've got an idea that I think could make this transformation a bit simpler, so I'll give that a go first :-)